modules/tests/linkchecker.py

```diff
@@ -199,25 +199,25 @@ def internal_external_link_checker(filepath, html_str):
                 # If problem detected, add to problem list
                 if links_list[link]:
                     if link not in problems:
-                        problems.append(link)
+                        problems.append(f"[other] {link}")
             elif link.startswith("http"):
                 # Consider status 404 and unreachable as broken.
                 # Unreachable will be triggered by the except clause
                 try:
                     r = requests.head(
                         link, headers=headers, 
                         verify=False, timeout=5)
-                    if r.status_code == 404:
+                    if r.status_code != 200:
```

Comment:
I remember some working links returning non 200 status codes like redirections which are 3XX

Reply:
That's still the case, but ideally this would prompt the admin/user/whomever to update those links to where the redirect points to. 

Another reason it's useful to catch 300 response codes here as well is because in some cases the 3XX status codes lead to 404 pages (which they conceptually probably shouldn't, but that's out of our control). If we didn't know to check 3XX pages we wouldn't be able to find those 404 errors with the test.